app.py

```diff
@@ -60,9 +60,16 @@ def get_meetup_events(group_list):
     group_apis = [i['field_events_api_key'] for i in group_list]
     # Create empty list to be returned by the function
     all_events = []
+
+    ## Meetup's API returns paginated results. These results have been limited to the first 50 events.
```

Comment:
This is my personal style, but I'd move the comment closer to the code it's about.

The 50 record and pagination is more applicable to line 72.

When you're doing comments you want to write for your future-self and future-someone else.

You will forget stuff about your own code. If the comment is 10 lines up you'll miss it and make the same mistake in 2 years.

Reply:
..and when I say "you", I mean, past-me.